lib/index.js

```diff
@@ -26,7 +26,8 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 module.exports = function(config) {
     var module = {};
-    
+
+    module.data = require('./data');
```

Comment:
For my OCD can you make the "= require" align with the others :)

Reply:
How about now?